webroot/src/components/Users/UserRowEdit/UserRowEdit.ts

```diff
@@ -109,6 +110,18 @@ class UserRowEdit extends mixins(MixinForm) {
         return this.getCompactPermission(this.currentUserCompactPermission) === Permission.ADMIN;
     }
 
+    get isCurrentUserStateAdminAny(): boolean {
+        const { currentUserCompactPermission } = this;
+        const statePermissions = currentUserCompactPermission?.states || [];
+        const isStateAdminAny = statePermissions.some((statePermission) => statePermission.isAdmin);
+
+        return isStateAdminAny;
+    }
+
+    get isAnyTypeOfAdmin(): boolean {
+        return this.isCurrentUserCompactAdmin || this.isCurrentUserStateAdminAny;
+    }
```

Comment:
Related to my prior comment: Can this code easily be abstracted away into the user object itself?

Reply:
Not easily at the moment - but hoping to make that the case after a walkthrough w/ product owner + #342 to handle the factoring once there is more clarity.

Reply:
Great!